integrations/defectdojo-JSONCreatedUsingSemgrepAPI/transformFindingsToJSON.py

```diff
@@ -0,0 +1,93 @@
+import requests
+import json
+import os
+from uploadSemgrepJSONToDefectDojo import uploadToDefectDojo
+import sys
+# Configuration
+deployment_slug = "swati"
+BASE_URL = "https://semgrep.dev"
+findings_url = BASE_URL+"/api/v1/deployments/"+deployment_slug+"/findings"
+SEMGREP_APP_TOKEN = os.environ["SEMGREP_API_TOKEN"]
+OUTPUT_FILE = "semgrep_findings_formatted.json"
+dd_url= "http://localhost:8080"
```

Comment:
I would recommend making these either CLI arguments or pulling them from the API via the provided Semgrep token (which should be SEMGREP_APP_TOKEN). We have some good examples of the latter in https://github.com/r2c-CSE/semgrep-utilities/blob/main/utilities/api/python_client_semgrep_api.py and I can point you to some CLI option examples too.